modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/roundstartslime.dm

```diff
@@ -1,11 +1,29 @@
 /datum/species/jelly
-	species_traits = list(MUTCOLORS,EYECOLOR,NOBLOOD,HAIR,FACEHAIR)
-	default_mutant_bodyparts = list("tail" = "None", "snout" = "None", "ears" = "None", "taur" = "None", "wings" = "None", "legs" = "Normal Legs", "horns" = "None",  "spines" = "None", "frills" = "None")
+	species_traits = list(
+		MUTCOLORS,
+		EYECOLOR,
+		NOBLOOD,
+		HAIR,
+		FACEHAIR,
+	)
+	default_mutant_bodyparts = list(
+		"tail" = "None",
+		"snout" = "None",
+		"ears" = "None",
+		"taur" = "None",
+		"wings" = "None",
+		"legs" = "Normal Legs",
+		"horns" = "None",
+		"spines" = "None",
+		"frills" = "None",
+	)
 	mutant_bodyparts = list()
 	hair_color = "mutcolor"
 	hair_alpha = 160 //a notch brighter so it blends better.
-	learnable_languages = list(/datum/language/common, /datum/language/slime)
-	payday_modifier = 0.75
```

Comment:
undocumented change-- atomization?

Reply:
Oh yeah I was gonna leave a comment about that but forgot, it's just redefining the variable to the same thing from the core file version of the race. This is not actually changing anything by being removed